templates/travel/components/utils/AmpListProps.js

```diff
@@ -0,0 +1,16 @@
+export default function AmpListProps(includeDates) {
+  return {
+    src:
+      '/api/search?maxPrice=0' +
```

Comment:
Use `URL` and `URL.searchParams` to create the string instead. It's much more readable.